cgi/product_multilingual.pl

```diff
@@ -1500,6 +1501,9 @@ ($product_ref, $field, $language)
 		# Notify robotoff
 		send_notification_for_product_change($product_ref, "updated");
 
+		# Create an event
+		send_event( { user_id => $User_id, event_type => "product_edited", barcode => $code});
```

Comment:
I find it a bit akward to specify points in the client that submit events. But it has the advantage of allowing us to change the points awarded for each event type without the change having a retroactive effect. It could also be used to have special operations like "double points on candies during Halloween!".

Reply:
I put 5. At this point it's only a first test, we can think of a better structure to define points awarded (maybe with rules like if the product has specific traits, then it's worth more points).

Reply:
yep, we could use the quality facet changed before / after edit, to give less or more points.
But I propose 5 in the sense that it at least reward more product edition than robotoff question answering. Fine tuning is for later :-) (and might require a game design expert :-P )